components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1530,6 +1530,14 @@ def getMaxPlaneSize(self):
                 int(c.getConfigValue('omero.pixeldata.max_plane_height')))
         return self._maxPlaneSize
 
+    def getRoiLimitSetting(self):
+        try:
+            roi_limit = (self.getConfigService().getConfigValue(
+                         "omero.client.viewer.roi_limit") or 100)
+        except:
+            roi_limit = 100
+        return roi_limit
```

Comment:
getConfigService().getConfigValue() - Does this return a string? If so, then you'd get a string returned from getRoiLimitSetting() if this worked, or 100 if it didn't. Should probably be a number in both cases?